Maps.md

```diff
@@ -0,0 +1,25 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas and an Area is a space with predefined boundaries and Things. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
```

Comment:
In fact, since this is three sentences (1 per concept), I think it *might* look nice as a list. Your call.